src/services/transaction.ts

```diff
@@ -23,13 +23,21 @@ async function cancel(id: string) {
 
   return res;
 }
-
-async function list(apiKey: string): Promise<IScheduledForUser[]> {
+// Promise<IScheduledForUser[]>
+async function list(apiKey: string, opts: any): Promise<[IScheduledForUser[], number]> {
```

Comment:
- define type for `opts`
- return object with clearly named properties instead of tuple. that way it's easier to understand what's going on